packages/rocketchat-slashcommand-tableflip/tableflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Tableflip is a named function that will replace /Tableflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Tableflip(command, params, item) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Tableflip" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2657732792/issues/source?bid=3199634&fileBranchId=3328556#l7)